### ISSUE TITLE
replace roboto-fontface with fontsource 

### ISSUE THREAD

**Issue by goapunk** [liqd/tempelhof#404]
see https://github.com/choffmeister/roboto-fontface-bower

**Comment by CarolingerSeilchenspringer** [comment]
@goapunk is this still relevant? if this is about the roboto font notice we got from client, then it is not relevant as long as we are not loading from external servers.

**Comment by goapunk** [comment]
no, it's just tht the package we use is no longer maintained. Generally that's ok as it's just a font. So we don't need to touch this if we don't plan another sprint on tempelhof

**Comment by CarolingerSeilchenspringer** [comment]
@goapunk I will leave this open in case plans change but we don't have to touch this now.
